README.md

```diff
@@ -47,7 +47,7 @@ libuv is licensed under the MIT license. Check the [LICENSE file](LICENSE).
 
 ## Community
 
- * [Mailing list](http://groups.google.com/group/libuv)
```

Comment:
I'd like to keep both. There are still questions coming from there. IMHO adding help is great, but I'd keep the ML until it has been silent to a long time.